Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,66 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-593-gc17fa86-src.tar.gz"
+  version "1.6.7-593"
+  sha256 "21579278c732674149bfe55a4f408c47f44f7d2df6bfc71f3b12f004aa14bd1b"
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
```

Comment:
I'm a bit "eh" about this style of `devel` to be honest. It means everyone who installs the `devel` build _could_ have slightly different builds and if it breaks and we get complaints we may end up debugging different commit points.

I'll let other maintainers chip in on this if they wish to.